### ISSUE TITLE
[tools/install.sh] Error in install process on CentOS: "projectname: unbound variable"

### ISSUE THREAD

**Issue by maltheism** [aces/Loris#6185]
**Describe the bug**
On CentOS i'm trying to run the install.sh script in tools and it fails because of the error:
```
[RuntimeException]                                                                     
  Source directory /var/www/Loris/vendor/facebook/webdriver has unpushed changes on the  
   current branch:                                                                       
  M	lib/WebDriverTargetLocator.php    
```

Also worth noting that the `composer update` command gives this output:
```
Package facebook/webdriver is abandoned, you should avoid using it. Use php-webdriver/webdriver instead.
Package zendframework/zend-diactoros is abandoned, you should avoid using it. Use laminas/laminas-diactoros instead.
Package phpunit/phpunit-mock-objects is abandoned, you should avoid using it. No replacement was suggested.

```

**Comment by johnsaigle** [comment]
This happens when you run `./install.sh`? I'm confused because the script doesn't use composer so it shouldn't have anything to do with `vendor/`.

**Comment by johnsaigle** [comment]
> zendframework/zend-diactoros

Diactoros was removed in #5894. Are you using the latest version of the code? Neither the latest `23.0-release` nor `master` contain diactoros.

**Comment by johnsaigle** [comment]
I think the linked PR should fix this issue.

**Comment by maltheism** [comment]
@johnsaigle I believe I was on minor by mistake. I'm getting another error on major.
```
---------------------------------------------------------------------
                   LORIS Installation Script
---------------------------------------------------------------------

PHP appears to be installed.

More information on the complete installation and setup process is available on the LORIS Wiki on GitHub.

Ready to continue? [yn] y


CentOS distribution detected.
Would you like to automatically set up your apache configuration files? (In development for CentOS) [yn] y
./install.sh: line 227: projectname: unbound variable
```

Anyway I'm going to try manually doing what the install.sh script does in the meantime. Thanks for the help!

**Comment by johnsaigle** [comment]
I think that has to do with some fuzzy bash code in install. `shellcheck` may be able to help.

**Comment by johnsaigle** [comment]
Resolved by #6196
